gravitee-apim-console-webui/src/management/configuration/configuration.route.ts

```diff
@@ -825,6 +825,24 @@ function configurationRouterConfig($stateProvider: StateProvider) {
         },
       },
     })
+    .state('management.settings.notification-settings', {
+      url: '/notification-settings',
+      component: 'environmentNotificationSettingsList',
+      data: {
```

Comment:
WDYT to Keep this param like previous routing configuration 
```suggestion
      data: {
           unauthorizedFallbackTo: 'management.home',
```

Reply:
Done